Assets/Engine/Tile/TileRework/Connections/AdjacencyTypes/OffsetAdjacency.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace SS3D.Engine.Tiles.Connections
+{
+    /// <summary>
+    /// Adjacency type used for objects that are not centred on a tile. Examples that use this are pipes (not the middle layer)
+    /// </summary>
+    [Serializable]
+    public struct OffsetConnector
+    {
+        public enum OffsetOrientation
+        {
+            o,
+            cNorth,
```

Comment:
the 'c' meshes in the adjacency connectors got renamed to 'u' in a previous PR, this seems to be another merge error. It seems that the AdvancedAdjacency.cs was merged correctly and uses 'u', this file didn't get that memo.

This should be 'uNorth'